README.md

```diff
@@ -40,29 +42,49 @@ To set-up the MVS, follow the steps below:
 
     `pip install -r requirements.txt`
 
-* Check if requirements were installed with
+* Download the [cbc-solver](https://projects.coin-or.org/Cbc) into your system from https://ampl.com/dl/open/cbc/ and integrate it in your system, ie. unzip, place into chosen path, add path to your system variables  (Windows: “System Properties” -->”Advanced”--> “Environment Variables”, requires admin-rights). 
+You can also follow the [steps](https://oemof.readthedocs.io/en/latest/installation_and_setup.html) from the oemof setup instructions
+
```

Comment:
I would put this where it was - I think then the PATH variables are updated automatically? If you do it while having your code editor open eg. Pycharm doesnt know where cbc is, afaik.

Reply:
Ok, fixed in f8e290c4ac8d6e7b5da8cff3364c66d7ce132007